docs/advanced-features/static-html-export.md

```diff
@@ -11,9 +11,11 @@ description: Export your Next.js app to static HTML, and run it standalone witho
   </ul>
 </details>
 
-`next export` allows you to export your Next.js application to static HTML, which can render without the need of a Node.js server. It is recommended to only use `next export` if you don't need any of the [unsupported features](#unsupported-features) requiring a server.
+Next.js can be used to generatic static applications, including using React in the browser without the need for a server.
```

Comment:
I don't think "without the need for a server" is correct.

What would serve the html besides a server?

Reply:
That's good feedback, agreed on saying Node.js specifically. Thank you for the suggestion.